Example/Views/ModalScreen.swift

```diff
@@ -44,6 +44,17 @@ enum ModalScreen {
                             textView.textFont = Font(name: "Helvetica")
                             textView.textSize = 18
                         }),
+                    imageView(
+                        image: Image(image: UIImage(named: "test")!),
```

Comment:
Use [this](https://github.com/guidomb/Portal/blob/4d411844c57d6652e9960bc2b3dd92fc50a3f2ec/Portal/View/UIKit/UIKitRenderable.swift#L21) method to load images

Reply:
What is the difference? if we expect the users to only use `loadImage` method, the image's init should be private?

Reply:
The main reason behind using `Image.loadImage` is not to couple user code with UIKit. Image init should be `internal`. In a future refactor we should change this, we should load image in a declarative view, loading the image should be done by the renderer.